doc/classes/AnimationPlayer.xml

```diff
@@ -94,6 +94,7 @@
 			<param index="0" name="name" type="StringName" />
 			<description>
 				Returns the first [AnimationLibrary] with key [param name] or [code]null[/code] if not found.
+				To get the [AnimationPlayer]'s global animation library, use [code]get_animation_library("")[/code].
```

Comment:
```suggestion
				To get the [AnimationPlayer]'s global animation library, use [method get_animation_library].
```

Reply:
`get_animation_library("")` was the purpose of this pull request, I feel it's a bit of a secret code to retrieve the global animation library.

Maybe this elaborates more:
```
To get the [AnimationPlayer]'s global animation library, use an empty string as an argument. Like so: [code]get_animation_library("")[/code].
```

Reply:
I think this is fine as is. I agree it does make sense to point out. :)